src/blog/lagom-1-5-0-M1.md

```diff
@@ -1,27 +1,47 @@
 ---
-title: Lagom 1.5.0-M1 released!
+title: Lagom 1.5.0-M2 released!
 date: 2018-07-25
 author_github: ignasi35
 tags: news
 summary: >
-    The Lagom team has released Lagom 1.5.0-M1
+    The Lagom team has released Lagom 1.5.0-M2
 ---
 
-We are happy to announce the availability of Lagom 1.5.0-M1. This is the first milestone towards the releasing of Lagom 1.5.0 which moves Lagom closer to Play and Akka. This release includes Play 2.7.0-M2 and starts moving towards supporting scala 2.13.
+We are happy to announce the availability of Lagom 1.5.0-M2 (unfortunately Lagom 1.5.0-M1 was DOA). This is the first milestone towards the releasing of Lagom 1.5.0 which moves Lagom closer to Play and Akka. This release includes Play 2.7.0-M2 and improves the cross-product development.
 
 The motivation behind moving Lagom closer to Play and Akka is twofold: Play and Akka users will be able to adopt Lagom more easily by reusing existing code or APIs they are already familiar with, and framework maintainers will reduce the amount of code to maintain and converge in conventions speeding up the development process.
 
+Note that this should not be used in production, that Lightbend Orchestration (sbt-reactive-app) does not yet support Lagom 1.5.0-M1, and that ConductR will not be supported for Lagom 1.5.0. Also note that this is a `Milestone` version and we don't guarantee API stability or binary compatibility between milestones.
```

Comment:
> Also note that this is a `Milestone` version

Backticks should be for code samples, identifiers, etc. You can use double quotes when trying to define a word or refer to a word, but in this case I think it's not necessary to quote it at all:

> Also note that this is a milestone version

Maybe could be clarified even further:

> Also note that this is an unsupported, pre-release milestone and we don't guarantee…